### DISCUSSION TITLE
SkyWalking UI return 405 Method Not Allowed V9.1

### DISCUSSION THREAD

**Discussion by CoffeeSite** [apache/skywalking#9265] [Q&A] [answered]
I deploy and configed skywakling, When I access Web UI got this error.
Is there any configuration  i missed?
![image](https://user-images.githubusercontent.com/3747121/175198697-cedc9dce-b320-406a-9160-e092cac6cc93.png)

**Comment by wu-sheng** [comment, marked as answer]
This usually means backend is not working.
